adoptopenjdk-api-v3-models/src/main/kotlin/net/adoptopenjdk/api/v3/models/MonthlyDownloadDiff.kt

```diff
@@ -0,0 +1,7 @@
+package net.adoptopenjdk.api.v3.models
+
+class MonthlyDownloadDiff(
+    val month: String,
```

Comment:
I would probably make this some proper data type, think YearMonth is probably right https://docs.oracle.com/javase/8/docs/api/java/time/YearMonth.html

Reply:
The resulting endpoint entry now looks like this:
```
"month": {
            "leapYear": true,
            "month": "MARCH",
            "monthValue": 3,
            "year": 2020
        },
```
Not sure if this is overkill for what we need?

Reply:
hmm, that is not a very nice object, yeah maybe we just go with "YYYY-MM" string